### PR TITLE
Add sql operations api to 2015-05-01-preview

### PR DESCRIPTION
**Pull request by jaredmoo**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jaredmoo** [comment]
@nathannfan fyi

File: specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/operations.json [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        "description": {
+          "description": "The localized friendly description for the operation.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
```
Comment (vishrutshah) [inline specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/operations.json]: minor: Seems like not used in this swagger

File: specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/operations.json [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The API version to use for the request.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "client"
+    },
+    "ResourceGroupParameter": {
```
Comment (vishrutshah) [inline specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/operations.json]: minor: seems like not used in this swagger

File: specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/operations.json [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+      "name": "resourceGroupName",
+      "in": "path",
+      "description": "The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ServerNameParameter": {
```
Comment (vishrutshah) [inline specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/operations.json]: minor: seems like not used :) Please remove all unused global parameters.

File: specification/sql/resource-manager/readme.md [hunk 4]
```diff
@@ -49,7 +49,6 @@ input-file:
 - Microsoft.Sql/2014-04-01/geoBackupPolicies.json
 - Microsoft.Sql/2014-04-01/importExport.json
 - Microsoft.Sql/2014-04-01/metrics.json
-- Microsoft.Sql/2014-04-01/operations.json
```
Comment (vishrutshah) [inline specification/sql/resource-manager/readme.md]: Why are we removing this reference without removing the files?

**Review by vishrutshah — CHANGES_REQUESTED** [review]
Changes LGTM! Just left few minor comments. Let me know your thoughts. Thanks!

File: specification/sql/resource-manager/readme.md (hunk 4 above)
Comment (jaredmoo) [reply specification/sql/resource-manager/readme.md]: 2014-04-01 operations is still supported, so it is still in swagger. However client code gen should use 2015-05-01-preview now.

File: specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/operations.json (hunk 1 above)
Comment (jaredmoo) [reply specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/operations.json]: You are right, but we are using Swashbuckle swagger generator and it's actually difficult to remove these unused params. Mind if we keep them?

**Comment by jaredmoo** [comment]
Feedback understood but I have no planned changes. Thanks Vishrut!

**Comment by vishrutshah** [comment]
Synced up offline with @jaredmoo. As the team is auto generating the swagger we may see the global `parameters` sections in each swagger but @jaredmoo has verified that this doesn't affect SDK generation.

**Review by vishrutshah — APPROVED** [review]
LGTM!

**Pull request merged**